lib/dartdoc.dart

```diff
@@ -11,6 +11,7 @@ library dartdoc;
 import 'dart:async';
 import 'dart:convert';
 import 'dart:io';
+import 'dart:mirrors';
```

Comment:
why was this added?

Reply:
leftover from an attempt to automatically read the version number from pubspec.yaml.